integrations/rabbitmq/prometheus_metadata.yaml

```diff
@@ -0,0 +1,128 @@
+platforms:
+  - type: GKE
+    detections:
+      - characteristic_metric:
+          metric_type: prometheus.googleapis.com/rabbitmq_connections/gauge
+    launch_stage: HIDDEN
+    exporter_metadata:
+      name: RabbitMQ Prometheus Exporter
+      doc_url: https://rabbitmq.com/prometheus.html#overview-prometheus
+      minimum_supported_version: v3.8.0
+    default_metrics:
```

Comment:
I noticed that the dashboard that uses this integration uses a rabbitmq_identity_info query. Is that something we should include in this list?